### DISCUSSION TITLE
ExcellentShop Support

### DISCUSSION THREAD

**Discussion by sharkaholic** [BG-Software-LLC/WildChests#195] [Pending Feature Requests]
Hi there! I see this has been requested in the past (about 2 years ago-ish!) and would like to give it a little BUMP!

ExcellentShop is an all-in-one economic plugin that allows us as admins to set prices for items and allows our players to sell their goods to the virtual store. It would be extremely helpful if we could see WildChests support ExcellentShop in terms of respecting the worth sat in the plugin and it's method of selling items!

Thanks in Advance!

**Comment by OmerBenGera** [comment]
Should be supported already.
If it doesnt work, open an issue regarding this.
